include/crm/common/scheduler.h

```diff
@@ -51,6 +51,114 @@ enum pe_quorum_policy {
+    pcmk_sched_quorate                  = (1ULL << 0),
+
+    //! Whether cluster is symmetric (via symmetric-cluster property)
+    pcmk_sched_symmetric_cluster        = (1ULL << 1),
+
+    //! Whether cluster is in maintenance mode (via maintenance-mode property)
+    pcmk_sched_in_maintenance           = (1ULL << 3),
+
+    //! Whether fencing is enabled (via stonith-enabled property)
+    pcmk_sched_fencing_enabled          = (1ULL << 4),
+
+    //! Whether cluster has a fencing resource (via CIB resources)
+    pcmk_sched_have_fencing             = (1ULL << 5),
+
+    //! Whether any resource provides or requires unfencing (via CIB resources)
+    pcmk_sched_enable_unfencing         = (1ULL << 6),
+
+    //! Whether concurrent fencing is allowed (via concurrent-fencing property)
+    pcmk_sched_concurrent_fencing       = (1ULL << 7),
+
+    /*!
+     * Whether resources removed from the configuration should be stopped (via
+     * stop-orphan-resources property)
+     */
+    pcmk_sched_stop_removed_resources   = (1ULL << 8),
+
+    /*!
+     * Whether recurring actions removed from the configuration should be
+     * cancelled (via stop-orphan-actions property)
+     */
+    pcmk_sched_cancel_removed_actions   = (1ULL << 9),
+
+    //! Whether to stop all resources (via stop-all-resources property)
+    pcmk_sched_stop_all                 = (1ULL << 10),
+
+    /*!
+     * Whether start failure should be treated as if migration-threshold is 1
+     * (via start-failure-is-fatal property)
+     */
+    pcmk_sched_start_failure_fatal      = (1ULL << 12),
+
+    //! \deprecated Do not use
+    pcmk_sched_remove_after_stop        = (1ULL << 13),
+
+    //! Whether unseen nodes should be fenced (via startup-fencing property)
+    pcmk_sched_startup_fencing          = (1ULL << 14),
+
+    /*!
+     * Whether resources should be left stopped when their node shuts down
+     * cleanly (via shutdown-lock property)
+     */
+    pcmk_sched_shutdown_lock            = (1ULL << 15),
+
+    /*!
+     * Whether resources' current state should be probed (when unknown) before
+     * scheduling any other actions (via the enable-startup-probes property)
+     */
+    pcmk_sched_probe_resources          = (1ULL << 16),
+
+    //! Whether the CIB status section has been parsed yet
+    pcmk_sched_have_status              = (1ULL << 17),
+
+    //! Whether the cluster includes any Pacemaker Remote nodes (via CIB)
+    pcmk_sched_have_remote_nodes        = (1ULL << 18),
+
+    // The remaining flags are scheduling options that must be set explicitly
+
+    /*!
+     * Whether to skip unpacking the CIB status section and stop the scheduling
+     * sequence after applying node-specific location criteria (skipping
+     * assignment, ordering, actions, etc.).
+     */
+    pcmk_sched_location_only            = (1ULL << 20),
+
+    //! Whether sensitive resource attributes have been masked
+    pcmk_sched_sanitized                = (1ULL << 21),
+
```

Comment:
Guess the reason for skipping a synonym for pe_flag_stdout is that it already isn't used anymore in pacemaker-code ...

Reply:
yep